<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hire</class>
 <widget class="QDialog" name="Hire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-37</y>
        <width>404</width>
        <height>445</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>CNIC</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>eMail</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>City</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_name"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_phone"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_cnic"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_email"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="le_country"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="le_city"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="le_address"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Job Title</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Salary</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Job Grade</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="le_job"/>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="sb_grade">
           <property name="maximum">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="db_salary">
           <property name="maximum">
            <double>99999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="cb_branch"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Branch</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="le_password"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_hire">
         <property name="text">
          <string>Hire</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
